### ISSUE TITLE
Aboud the learning rate?

### ISSUE THREAD

**Issue by idealboy** [nmhkahn/CARN-pytorch#28]
Sir,

Should it be:

176     def decay_learning_rate(self):
177         lr = self.cfg.lr * (0.5 ** (self.step / self.cfg.decay))
178         return lr

or:

176     def decay_learning_rate(self):
177         lr = self.cfg.lr * (0.5 ** (self.step // self.cfg.decay))
178         return lr

thank you very much!

eg:

110 // 130 =0
110 / 130 = 0.8461538461538461
